miniGithub/views.py

```diff
@@ -1,3 +1,31 @@
 from django.shortcuts import render
 
 # Create your views here.
+
+from django.shortcuts import render
+from django.contrib.auth import login, authenticate
+from django.contrib.auth.forms import UserCreationForm
+from django.shortcuts import render, redirect
+from .forms import SignUpForm
+from django.urls import reverse
+
+def home_view(request):
+    return render(request, 'miniGithub/home.html')
+
+def signup_view(request):
+    form = SignUpForm(request.POST)
+    if form.is_valid():
+        user = form.save()
+        user.refresh_from_db()
+        user.profile.first_name = form.cleaned_data.get('first_name')
+        user.profile.last_name = form.cleaned_data.get('last_name')
+        user.profile.email = form.cleaned_data.get('email')
+        user.save()
+        username = form.cleaned_data.get('username')
+        password = form.cleaned_data.get('password1')
+        user = authenticate(username=username, password=password)
+        login(request, user)
```

Comment:
Mislim da posle registracije nismo specificirali automatski login na aplikaciju, ali super sto si to implementirao